infrastructure/terraform/main.tf

```diff
@@ -0,0 +1,68 @@
+resource "azurerm_resource_group" "main" {
+  name     = local.resource_group_name
+  location = local.region
+}
+
+module "app-key-vault" {
```

Comment:
Once this key vault exists, is the idea that we would log in via the azure portal and do our secrets management there (e.g. when adding new secrets, or rotating a secret)?

Reply:
Yes indeed. I've now added a section to the readme to explain it. Please take a look.

Reply:
Ta, looks good 👍🏻